bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -146,17 +147,11 @@
                 optional_attributes={
                   'loading': 'eager',
                   'alt': 'Firefox',
-                  'class': 'firefox-logo'
-                })
-              }}
-              {{ resp_img(
-                url='img/firefox/challenge-the-default/logo-ctd.svg',
-                optional_attributes={
-                  'loading': 'eager',
-                  'alt': 'Challenge the default',
-                  'class': 'ctd-logo'
+                  'class': 'firefox-logo',
+                  'width': '50px'
                 })
               }}
+              <div class="ctd-logo-sprite"></div>
```

Comment:
Same as above for the mobile. banner, suggest adding `Challenge the Default` to the Firefox logo `alt` text?